admin/src/components/Processors/EditProcessor.test.js

```diff
@@ -30,8 +37,9 @@ describe('(Component) Edit Processor Form', () => {
     const onSubmit = jest.fn();
     const { component } = setupShallow({ onSubmit });
 
-    const form = component.find('form').at(0);
-    form.simulate('submit');
-    expect(onSubmit).toHaveBeenCalled();
+    const button = component.find('button[type="submit"]');
+    console.log(button.debug());
+    // button.simulate('click');
```

Comment:
looks unfinished, weren't you able to make it work with informed and the new lack of `<form>`?